geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/CommonParams.scala

```diff
@@ -43,8 +43,11 @@ trait OptionalTypeNameParam extends TypeNameParam {
 }
 
 trait PasswordParams {
-  @Parameter(names = Array("-p", "--password"), description = "Connection password")
+  @Parameter(names = Array("-p", "--password"), description = "Connection password (will prompt if not supplied and not using Kerberos)")
   var password: String = null
+
+  @Parameter(names = Array("--keytab"), description = "Path to Kerberos keytab file")
+  var keytab: String = null
```

Comment:
Yes, I've already done that to avoid the conflict with master. Just fixing up the test which fail due to the stricter Params checking even when using mock